docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
+    build: .
```

Comment:
wath's about moving the Dockerfile in a subdirectory to keep the root folder clean?